### PR TITLE
JDK-8328930: [AIX] remove pase related coding

### PR DESCRIPTION
**Pull request by MBaesken**
We still have quite a lot of OS400 / pase related coding in the hotspot AIX codebase. But this does not work and was never supported in OpenJDK. So we can remove this.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8328930](https://bugs.openjdk.org/browse/JDK-8328930): [AIX] remove pase related coding (**Bug** - P4)

### Reviewers
 * [Christoph Langer](https://openjdk.org/census#clanger) (@RealCLanger - **Reviewer**)
 * [Lutz Schmidt](https://openjdk.org/census#lucy) (@RealLucy - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18471/head:pull/18471` \
`$ git checkout pull/18471`

Update a local copy of the PR: \
`$ git checkout pull/18471` \
`$ git pull https://git.openjdk.org/jdk.git pull/18471/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18471`

View PR using the GUI difftool: \
`$ git pr show -t 18471`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18471.diff">https://git.openjdk.org/jdk/pull/18471.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18471#issuecomment-2018014068)

### CODE REVIEW

File: src/hotspot/os/aix/os_aix.cpp [hunk 1]
```diff
@@ -619,66 +602,42 @@ void os::init_system_properties_values() {
...
-    unsigned long long pgsp_free = 0;
-    if (libo4::get_memory_info(&virt_total, &real_total, &real_free, &pgsp_total, &pgsp_free)) {
-      pmi->virt_total = virt_total;
-      pmi->real_total = real_total;
-      pmi->real_free = real_free;
-      pmi->pgsp_total = pgsp_total;
-      pmi->pgsp_free = pgsp_free;
-      return true;
-    }
+  // On AIX, I use the (dynamically loaded) perfstat library to retrieve memory statistics
```
Comment (RealCLanger) [inline src/hotspot/os/aix/os_aix.cpp:607]: maybe remove the "I" pronoun in the comment here? Also the links could be placed on one line.

File: src/hotspot/os/aix/os_aix.cpp [hunk 2]
```diff
@@ -619,66 +602,42 @@ void os::init_system_properties_values() {
...
-    // u_longlong_t pgsp_free          Free paging space (in 4 KB pages).
+  assert(rc == 1, "perfstat_memory_total() - weird return code");
 
-    pmi->virt_total = psmt.virt_total * 4096;
-    pmi->real_total = psmt.real_total * 4096;
-    pmi->real_free = psmt.real_free * 4096;
-    pmi->pgsp_total = psmt.pgsp_total * 4096;
-    pmi->pgsp_free = psmt.pgsp_free * 4096;
+  // excerpt from
+  // http://publib.boulder.ibm.com/infocenter/systems/index.jsp
```
Comment (RealCLanger) [inline src/hotspot/os/aix/os_aix.cpp:626]: Same here. URL on one line. Does it still work?

**Review by RealCLanger — APPROVED** [review]
Welcome cleanup. I made two minor comments.

File: src/hotspot/os/aix/os_aix.cpp (hunk 2 above)
Comment (MBaesken) [reply src/hotspot/os/aix/os_aix.cpp:626]: The link is dead, it does not work any more. I think we can remove it.

**Comment by MBaesken** [comment]
Hi Christoph, I adjusted/removed the comments.

**Review by RealCLanger — APPROVED** [review]

**Review by RealLucy — APPROVED** [review]
Looks good.

**Comment by MBaesken** [comment]
Hi Lutz and Christoph, thanks for the reviews !

/integrate

**Pull request closed without merging**